public_html/wp-content/plugins/wordcamp-speaker-feedback/includes/post.php

```diff
@@ -0,0 +1,66 @@
+<?php
```

Comment:
Two reasons:

1. The `post_accepts_feedback` function is also used in the REST API controller. Since it has multiple contexts, it seemed better to keep it separated from either context. That helps avoid the code making assumptions that are too specific to one particular context.
1. It seems possible that we will eventually have other functions that deal specifically with the posts that feedback comments are attached to. When I wrote this PR, I had at least one other in mind, but now I'm not thinking of it 😕

Reply:
Makes sense! thanks for answering my curiosity 🙂